src/Fixer/ClassNotation/SelfStaticAccessorFixer.php

```diff
@@ -29,7 +31,7 @@ final class SelfStaticAccessorFixer extends AbstractFixer
     public function getDefinition(): FixerDefinitionInterface
     {
         return new FixerDefinition(
-            'Inside a `final` class or anonymous class `self` should be preferred to `static`.',
+            'Inside a `final` class, anonymous class, or enums, `self` should be preferred over `static`.',
```

Comment:
Maybe like this? With so many commas it looks somehow wrong.

```suggestion
            'Inside an enum or `final`/anonymous class `self` should be preferred over `static`.',
```